releases/fuji/compose-files/docker-compose-fuji-1.0.0.yml

```diff
@@ -0,0 +1,579 @@
+#  * Copyright 2019 Intel Corporation
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  * @author: Jim White, Dell
+#  * added: July 26, 2019
+#  * @author: Jim Wang, Intel Corp
+#  * updated: October 8, 2019
```

Comment:
The authors here don't match the commit authors? At the very least @tingyuz should be included in the list here

Reply:
yes, @tingyuz  please add yourself as well

Reply:
Fixed.